app/integration_action.go

```diff
@@ -325,27 +325,34 @@ func (w *LocalResponseWriter) WriteHeader(statusCode int) {
 	w.status = statusCode
 }
 
-func (a *App) DoLocalRequest(rawURL string, body []byte) (*http.Response, *model.AppError) {
+func (a *App) doPluginRequest(method, rawURL string, values url.Values, body []byte) (*http.Response, *model.AppError) {
```

Comment:
I am a little confused - is this a relevant change, or a rebase/merge issue? This file doesn't seem relevant to autocomplete.

Reply:
It is relevant for downloading dynamic data from plugins. I just reorganized code so we could have different http methods `GET` , `POST`..